proposals/NNNN-ServiceStatusUpdateToHMI.md

```diff
@@ -0,0 +1,114 @@
+The motivation behind this proposal is to over come this issue by introducing a new notification between SDL and HMI.
+
+
+## Proposed solution
+
+
+### Additions to HMI_API
+
+The proposed solution is to introduce a new RPC for SDL Core to update HMI with the status of a particular service.
+
+```xml
+<function name="OnServiceUpdate" messagetype="notification">
+  <description>
+    Must be sent by SDL to HMI when there is an update on status of certain services.
+    Services supported with current version: Video
+  </description>
+  <param name="serviceType" type="Common.ServiceType" mandatory="true">
+    <description>Specifies the service which has been updated.</description>
+  </param>
+  <param name="serviceEvent" type="Common.ServiceEvent" mandatory="false">
+    <description>Specifies service update event.</description>
+  </param>
+  <param name="reason" type="Common.ServiceUpdateReason" mandatory="false">
+    <description>
+      The reason for a service event. Certain events may not have a reason such as when a service is ACCEPTED (which is the normal expected behavior).
+    </description>
+  </param>
+</function>
+
+<interface name="Common" version="2.0.0" date="2018-09-05">
+:
+:
+  <enum name="ServiceType">
+    <element name="VIDEO" >
+      <description>Refers to the Video service.</description>
+    </element>
+  </enum>
+
+  <enum name="ServiceEvent">
+    <element name="REQUEST_RECEIVED" >
+      <description>When a request for a Service is received.</description>
+    </element>
+    <element name="REQUEST_ACCEPTED" >
+      <description>When a request for a Service is Accepted.</description>
+    </element>			
+    <element name="REQUEST_REJECTED" >
+      <description>When a request for a Service is Rejected.</description>
+    </element>			
+  </enum>	
+
+  <enum name="ServiceUpdateReason">
+    <element name="PTU_FAILED" >
+      <description>When a Service is rejected because the system was unable to get a required Policy Table Update.</description>
+    </element>
+    <element name="INVALID_CERT" >
+      <description>When a Service is rejected because the security certificate is invalid/expired.</description>
+    </element>			
+    <element name="INVALID_TIME" >
+      <description>When a Service is rejected because the system was unable to get a valid SystemTime from HMI which is required for certificate authentication.</description>
```

Comment:
Change to:
When a Service is rejected because the system was unable to get a valid SystemTime from HMI, which is required for certificate authentication.